### ISSUE TITLE
Optimize internal representation and comparison operation

### ISSUE THREAD

**Issue by fabiolimace** [f4b6a3/ksuid-creator#9]
1. Replace the internal field `byte[20] bytes` with two fields: `int seconds` and `byte[16] payload`.
2. Optimize the `Ksuid.compareTo()` method after the change in the internal representation.

**Comment by fabiolimace** [comment]
1. The internal field `byte[20] bytes` was replaced by two fields: `int seconds` and `byte[16] payload`.

2. Optimization: the `Ksuid.compareTo()` method is 30% faster.

BEFORE:

```
Benchmark                        Mode  Cnt     Score     Error   Units
Ksuid_compareTo   (*)           thrpt    5  6318,844 ±  48,798  ops/ms 
Ksuid_fromString                thrpt    5  3387,946 ±  34,050  ops/ms
Ksuid_toString                  thrpt    5  2147,112 ±  37,902  ops/ms
KsuidCreator_getKsuid           thrpt    5  1918,965 ±  52,291  ops/ms
KsuidCreator_getKsuid_toString  thrpt    5  1015,992 ±  27,793  ops/ms
```

AFTER:

```
Benchmark                        Mode  Cnt     Score     Error   Units
Ksuid_compareTo   (*) (+30%)    thrpt    5  8276,152 ±  47,357  ops/ms
Ksuid_fromString                thrpt    5  3509,903 ±  57,578  ops/ms
Ksuid_toString                  thrpt    5  2176,286 ±  30,880  ops/ms
KsuidCreator_getKsuid           thrpt    5  1929,631 ±  32,927  ops/ms
KsuidCreator_getKsuid_toString  thrpt    5  1012,919 ±  17,240  ops/ms
```
(*) The Ksuid.compareTo() method was executed 100 times per benchmark operation.

**Comment by fabiolimace** [comment]
Released version v2.2.0.
